crates/anemo/src/config.rs

```diff
@@ -561,6 +563,7 @@ impl EndpointConfig {
         &self.quinn_client_config
     }
 
+    #[allow(deprecated)]
```

Comment:
maybe add a comment here on why this is necessary and/or a TODO to remove once upgrading rustls version?